src/components/Login/Login.js

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
 import GoogleLogin from 'react-google-login';
 import * as PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
 import { failLogin, successLogin } from '../../services/loginService';
 
+const useStyles = makeStyles(() => ({
+  root: {
+    width: '100%',
+    height: '100%',
+    margin: 'auto 50%'
+  }
+}));
+
 const Login = ({ setAuth }) => {
+  const classes = useStyles();
+
   const success = (res) => {
     successLogin(res);
     setAuth(true);
   };
 
   return (
-    <div className="login">
+    <div className={classes.root}>
```

Comment:
Cool! I think I can see that we are using makeStyles from material ui. We can now use this practice for the rest of the CSS refactoring next week! Great!